### ISSUE TITLE
Implement Collapse

### ISSUE THREAD

**Issue by Utkarsh4517** [opxica/modular-ui#89]
Inspiration :
https://www.material-tailwind.com/docs/react/collapse

**Comment by deepraj02** [comment]
Hey assign me this one, if not done

**Comment by Utkarsh4517** [comment]
yeah sure go ahead. @deepraj02 just checkout our docs at https://www.dub.sh/modularui-docs to get a better understanding of our practices.
